### ISSUE TITLE
Source map error

### ISSUE THREAD

**Issue by loescornelis** [Turfjs/turf#2721]
After updating to version 7.1.0 (From 6.5.0) we get the following error:

WARNING in ../../node_modules/@turf/jsts/dist/jsts.min.js
Module Warning (from ../../node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@turf/jsts/dist/jsts.min.js.map' file: Error: ENOENT: no such file or directory, open '//node_modules/@turf/jsts/dist/jsts.min.js.map'

Could the sourcemaps be included?

**Comment by VadimCiv** [comment]
Same here: Turf 7.1.0 with React 18.
Has anyone fixed this issue?

```
[REACT] WARNING in ./node_modules/@turf/jsts/dist/jsts.min.js
[REACT] Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
[REACT] Failed to parse source map from './node_modules/@turf/jsts/dist/jsts.min.js.map' file: Error: ENOENT: no such file or directory, open './node_modules/@turf/jsts/dist/jsts.min.js.map'
```

**Comment by smallsaucepan** [comment]
Hi @loescornelis and @VadimCiv. Is this just a warning displayed in dev mode? Or is it affecting the functionality of your app?

I ask because I don't believe @turf/jsts *ever* included a map file.

**Comment by VadimCiv** [comment]
@smallsaucepan

> displayed in dev mode, IDK if affecting the functionality

**Comment by smallsaucepan** [comment]
Ok thanks. Will wait to see if @loescornelis is having a different experience. If it isn't affecting functionality though, it will be pretty low priority.

It's possible the warning has always been there and source-map-loader has only recently started noticing it.

**Comment by loescornelis** [comment]
@smallsaucepan 

As far as I can tell it's not affecting functionality. We've managed to hide the warning for now :+1:

**Comment by smallsaucepan** [comment]
@loescornelis @VadimCiv a new version of @turf/jsts has been released (2.7.2) which has a map file. Next time you build / deploy would you mind seeing if the new version has been picked up and the warning fixed? Thanks!

**Comment by brenwell** [comment]
Works for me!

**Comment by loescornelis** [comment]
This has solved the problem for us as well.
Thank you!!

**Comment by VadimCiv** [comment]
> @loescornelis @VadimCiv a new version of @turf/jsts has been released (2.7.2) which has a map file. Next time you build / deploy would you mind seeing if the new version has been picked up and the warning fixed? Thanks!

Thank you for the update! Great job!
